### ISSUE TITLE
ofCamera / ofEasyCam - add easy way to save/load settings.

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/openFrameworks#1683]
I spent a few hours trying to find which data within ofEasyCam needed to be load / saved to disk to save and restore a view.  

I found it pretty unclear ( due to the multiple levels of inheritance ), where this core data lies. 

In the end I resorted to memcpy-ing the entire object to a buffer and writing it to disk as a binary file. 
This works well for my needs - but maybe we could make it a little easier? :)

This could relate to ofParameter and ofParamList->serialize() or something like that. 

@elliotwoods @arturoc

**Comment by mrbichel** [comment]
Has there been any more work on this.e.g. Making ofBaseSerializer work with ofCamera and ofEasyCam?

**Comment by arturoc** [comment]
no, not yet, i know @tgfrerer is experimenting at the moment with some serializing libraries and it'll probably be good to add support for something like that throughout the core so any object can be serialized.

**Comment by roymacdonald** [comment]
Hi @mrbichel, I don't think so. It seems that the option for saving/loading ofCamera is the addon I made some time ago.

**Comment by neilmendoza** [comment]
What about making position, orientation and scale protected in ofNode or adding them to a protected ofParameterGroup? Being able to derive classes from ofNode that can access these parameters would be useful.

**Comment by roymacdonald** [comment]
@neilmendoza 👍 
I think that I have made these `protected` a few times for using on some projects. It happens to be quite useful.
Is there any reason for having these as `private`?
